sdk/communication/Azure.Communication.Common/src/CommunicationCloudEnvironment.cs

```diff
@@ -19,13 +19,22 @@ public CommunicationCloudEnvironment(string value)
         private const string PublicValue = "public";
         private const string DodValue = "dod";
         private const string GcchValue = "gcch";
+        private const string AirGap08Value = "airgap08";
+        private const string AirGap09Value = "airgap09";
+        private const string GallatinValue = "gallatin";
```

Comment:
Can you please get a confirmation from an authority that this list is complete (to this date)?

Reply:
Confirmed with Auth team. These clouds won't be supported.